scripts/wait-for-cluster-ready.sh

```diff
@@ -6,9 +6,10 @@ CLUSTER_NAME="$1"
 REGION="$2"
 export AWS_DEFAULT_REGION=${REGION}
 echo "Inputs CLUSTER_NAME  : ${CLUSTER_NAME} , REGION: ${REGION} "
-BIN_DIR=$(cat .bin_dir)
-cat .bin_dir
 
+# if [[ -n "${BIN_DIR}" ]]; then
```

Comment:
Same here... if not needed, can we get rid of these commented sections?

Reply:
Removed comments